### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Andrew77788**

### CODE REVIEW

File: src/convertToObject.js [hunk 1]
```diff
@@ -6,7 +6,18 @@
...
+  const str = sourceString.split(';');
+  const result = {};
+
+  str.map((element) => {
+    if (element.length >= 2) {
+      const splitElement = element.split(':').map((item) => item.trim());
+
+      result[splitElement[0]] = splitElement[1];
+    }
+  });
```
Comment (anastasiia-tilikina) [inline src/convertToObject.js:18]: it is not a proper usage for map method. Map should return some array. Use either forEach of reduce method here

**Review by anastasiia-tilikina — CHANGES_REQUESTED** [review]
Let's fix some issue here

File: src/convertToObject.js [hunk 2]
```diff
@@ -6,7 +6,19 @@
...
 function convertToObject(sourceString) {
-  // write your code here
+  const str = sourceString.split(';');
+  const result = {};
+
+  str.forEach((element) => {
+    if (element.length >= 2) {
+      const splitElement = element.split(':').map((item) => item.trim());
+
+      result[splitElement[0]] = splitElement[1];
```
Comment (volodymyr-soltys97) [inline src/convertToObject.js:16]:
```suggestion
      const [property, value] = element.split(':').map((item) => item.trim());

      result[property] = value;
```

**Review by volodymyr-soltys97 — APPROVED** [review]
Good job 👍

**Pull request closed without merging**